include/GoogleSync/GoogleSync.php

```diff
@@ -619,9 +741,8 @@ public function pushPullSkip($event_1, $event_2 = null)
      *
      * @return bool Success/Failure
      */
-    public function pushEvent($event_local, $event_remote = null)
+    public function pushEvent(Meeting $event_local, Google_Service_Calendar_Event $event_remote = null)
     {
-
         if (!isset($event_remote) || empty($event_remote)) {
             $event = $this->createGoogleCalendarEvent($event_local);
```

Comment:
here and above (https://github.com/salesagility/SuiteCRM/pull/6539/files#diff-034e880ea433e2cc2571088f7cbc6620R750) returned value could be false as failure. Please handle the error.

Reply:
Fixed same as above